config/initializers/content_security_policy.rb

```diff
@@ -4,7 +4,7 @@
 
 # rubocop:disable Metrics/BlockLength
 Rails.application.config.content_security_policy do |policy|
-  connect_src = ["'self'"]
+  connect_src = ["'self'", "https://verify.socure.us"]
```

Comment:
A couple questions:

- Would we want to make the URL configurable?
- Should this be changing `form_action` instead?

Regardless of the above, I think we'd want to limit the scope of when the extra allowances are made to where it's needed with something like `override_form_action_csp` in the relevant controller(s).

Reply:
Yes and maybe. There's actually a key in `application.yml` we should extract this from, I think.
I agree that careful review of the CSP is in order; right now, this branch is *do not merge* because we need to either think those changes through carefully or not change anything and accept that Socure handoff won't work until the CSP is addressed. I vote for the latter; this change is just to verify that it really is the CSP, and changing it can work. It is and does.